en/backups.md

```diff
@@ -0,0 +1,660 @@
+    of the server. Change *backupserver* to the actual host name.
+
+    Also, make sure the *Password* directive matches the password string in
+    `/etc/bacula/bacula-sd.conf`.
+
+-   Create a new *FileSet*, which will determine what directories to backup,
+    by adding:
+
+        # LocalhostBacup FileSet.
+        FileSet {
+          Name = "LocalhostFiles"
+          Include {
+            Options {
+              signature = MD5
+              compression=GZIP
+            }
+            File = /etc
+            File = /home
+          }
+        }
+
+    This *FileSet* will backup the `/etc` and `/home` directories. The
+    *Options* resource directives configure the FileSet to create an MD5
+    signature for each file backed up, and to compress the files using GZIP.
+
+-   Next, create a new *Schedule* for the backup job:
+
+        # LocalhostBackup Schedule -- Daily.
+        Schedule {
+          Name = "LocalhostDaily"
+          Run = Full daily at 00:01
+        }
+
+    The job will run every day at 00:01 or 12:01 am. There are many other
+    scheduling options available.
+
+-   Finally create the *Job*:
+
+        # Localhost backup.
+        Job {
+          Name = "LocalhostBackup"
+          JobDefs = "DefaultJob"
+          Enabled = yes
+          Level = Full
+          FileSet = "LocalhostFiles"
+          Schedule = "LocalhostDaily"
+          Storage = TapeDrive
+          Write Bootstrap = "/var/lib/bacula/LocalhostBackup.bsr"
+        }  
+
+    The job will do a *Full* backup every day to the tape drive.
+
+-   Each tape used will need to have a *Label*. If the current tape does not
+    have a label Bacula will send an email letting you know. To label a tape
+    using the Console enter the following from a terminal:
+
+        bconsole
+
+-   At the Bacula Console prompt enter:
+
+        label
+
+-   You will then be prompted for the *Storage* resource:
+
+
+        Automatically selected Catalog: MyCatalog
+        Using Catalog "MyCatalog"
+        The defined Storage resources are:
+             1: File
+             2: TapeDrive
+        Select Storage resource (1-2):2
+
+-   Enter the new *Volume* name:
+
+
+        Enter new Volume name: Sunday
+        Defined Pools:
+             1: Default
+             2: Scratch
+
+    Replace *Sunday* with the desired label.
+
+-   Now, select the *Pool*:
+
+
+        Select the Pool (1-2): 1
+        Connecting to Storage daemon TapeDrive at backupserver:9103 ...
+        Sending label command for Volume "Sunday" Slot 0 ...
+
+Congratulations, you have now configured *Bacula* to backup the localhost to
+an attached tape drive.
+
+## Resources {#bacula-resources}
+
+-   For more *Bacula* configuration options, refer to [Bacula's
+    Documentation].
+
+-   The [Bacula Home Page] contains the latest Bacula news and developments.
+
+-   Also, see the [Bacula Ubuntu Wiki] page.
+
+  [???]: #network-file-system
+  [References]: #backup-shellscript-references
```

Comment:
Most of the links look good, but there are occasional [???] entries on pages. Also, should we explicitly add the .html suffexes to the internal links?